core/src/main/kotlin/org/evomaster/core/problem/graphql/service/GraphQLFitness.kt

```diff
@@ -238,6 +237,8 @@ open class GraphQLFitness : HttpWsFitness<GraphQLIndividual>() {
             val bugId = idMapper.handleLocalTarget(descriptiveId)
             fv.updateTarget(bugId, 1.0, indexOfAction)
 
+        } else if (DefinedFaultCategory.HTTP_STATUS_500 !in config.getDisabledOracleCodesList()) {
```

Comment:
wouldn't this be printed each time we have a status different from 500? also, to avoid flooding the logs, could be in a `LoggingUtil.uniqueUserInfo`. Also the message could rather be: `Found endpoints with status code 500. But those are not marked as fault,  as HTTP 500 fault detection has been disabled"`